virtualbox/create-vm.sh

```diff
@@ -10,6 +10,8 @@ PAE="on";
 VRAM=8;
 USB="off";
 
+readonly IP_ADDRESS=192.168.99.50
```

Comment:
We should perhaps think about passing this IP address as parameter but then the code should be changed in order to configure the dhcp server with the corresponding netmask, network. WDYT ?

Reply:
That is a good idea and it's definitely doable, but it would take some extra work.
I propose that we create a separate issue for that and work on it if/when we have some spare time, since it's not critical at the moment.